packages/salesforcedx-vscode-soql/src/lspClient/orgMetadata.ts

```diff
@@ -47,11 +47,26 @@ export class FileSystemOrgDataSource implements OrgDataSource {
     const standardsFolder = path.join(soqlMetadataPath, STANDARDOBJECTS_DIR);
 
     const files: string[] = [];
-    if (fs.existsSync(standardsFolder)) {
-      files.push(...(await fs.promises.readdir(standardsFolder)));
+    try {
+      const standardsUri = vscode.Uri.file(standardsFolder);
+      const standardsStat = await vscode.workspace.fs.stat(standardsUri);
```

Comment:
what's the value of the stat here?  Won't the `catch` block handle the "doesn't exist" scenario if readdir fails?

Reply:
You're right